packages/pwa-kit-runtime/src/utils/ssr-config.server.js

```diff
@@ -52,7 +59,7 @@ export const getConfig = () => {
     // Match config files based on the specificity from most to most general.
     const explorerSync = cosmiconfigSync(targetName, {
         packageProp: 'mobify',
-        searchPlaces: searchPlaces.map((path) => (isRemote ? `build/${path}` : path)),
+        searchPlaces: searchPlaces,
```

Comment:
This logic is not required anymore as the same result is accomplished by using the `searchFrom` property in the search function below. (this option was also required to fix the original issue and look in a different folder).